cli/config.go

```diff
@@ -48,15 +43,5 @@ func storeConfigToCache(cfg *Config) error {
 
 // Config contains stored config information for the CLI.
 type Config struct {
-	Auth      string `json:"auth"` // opaque
-	AuthEmail string `json:"-"`
-}
-
-func (c *Config) parseAuthInfo() error {
-	var claims rpc.JWTClaims
-	if _, _, err := jwt.NewParser().ParseUnverified(c.Auth, &claims); err != nil {
-		return err
-	}
-	c.AuthEmail = claims.AuthMetadata["email"]
-	return nil
+	Auth *Token `json:"auth"`
```

Comment:
how will existing CLIs fare once we merge this?

Reply:
It will break right now. Do you think we need to support backwards compatibility given the low usage of the CLI? Happy to add some logic to handle it.

Reply:
nah, just wanted to understand

Reply:
ack